joss/paper.md

```diff
@@ -0,0 +1,163 @@
+laboratories. The focus of this work is neutron stars, thought to be the most
+compact extended objects in the Universe. Observable radiation from a neutron
+star encodes information about fundamental physics (gravity, electromagnetism,
+and nuclear forces) and astrophysical processes (such as the state and evolution
+of a stellar magnetosphere). Neutron stars are often detected by astronomers
+because their radiative signal - in radio, X-ray and gamma-rays -  is pulsed due to bulk stellar rotation.  By modeling the physical process that generates data registered by telescopes, astronomers and astrostatisticians can make inferential statements
+about the nature of the extreme Universe.
+
+
+# Statement of need
+
+There exist open-source libraries and packages to support a subset of the
+modeling treated in astrophysical literature. They provide frameworks,
+toolsets, model implementations, and so on. One sub-field for which there does
+not exist such an open-source project, is for the *statistical modeling* of
+X-ray signals that pulse due to rotational modulation of asymmetric emission
+from the surface of a neutron star. Pulsing X-ray signals from neutron stars
+are modeled to statistically estimate parameters such as stellar mass and
+radius, and properties of the surface radiation field such as a map of
+temperature. The mass and radius of a neutron star are a function of the
+equation of state of internal matter (especially the dense matter in the core)
+and the formation history of the star (that determines the central energy
+density and the spin frequency); the state of the surface radiation field is
+the product of a potentially long and complex stellar evolutionary history,
+especially that of the stellar magnetosphere. Such parameter estimation
+requires relativistic tracing of radiation as it propagates from surface to a
+distant telescope.
+
+There are a small number of open-source libraries for simulating the X-ray
+signals from rapidly spinning neutron stars and more generally from the
+vicinity of general relativistic compact objects (including black holes), but
+the scope of these projects does not include statistical modeling, which
+necessitates tractable parametrised models and a modular framework for
+constructing those models. The state-of-the-art example is the Arcmancer
+library of @Pihajoki:2018, a general purpose toolbox that is wrapped by an
+updated version of the bender library of @Nattila:2016 for the purpose of
+simulating X-ray signals from hot regions on the surfaces of rapidly rotating
+neutron stars. These general purpose libraries have a wide array of
+applications and would exhibit synergy with a library for statistical modeling
+of X-ray signals registered by space telescopes. The general purpose libraries
+are for example useful for accurate simulations of realistic, general
+relativistic neutron stars. A statistical modeling library would be useful for
```

Comment:
Same here. Why would? Aren't we introducing exactly that library here?

Reply:
Updated text.